scripts/update_node.py

```diff
@@ -40,6 +40,7 @@
       shutil.move(dirname, 'bin')
       os.mkdir(dirname)
       shutil.move('bin', dirname)
+      os.remove(filename)
```

Comment:
doesn't this break the test?

Reply:
This fixed a bug where we were including two copies of node in windows.  We were calling zip with the existing zip file present which was adding the new "bin" directory, but not removing the old zip contents.   So it saves a bit of space for all windows users.